meilisearch/models.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from typing import Any, List, Dict, Optional
+from camel_converter.pydantic_base import CamelBase
+
+@dataclass
+class MultiDocument:
+    results: List[Dict[str, Any]]
+    offset: int
+    limit: int
+    total: int
+
+class Task(CamelBase):
+    uid: Optional[str]
+    index_uid: str
+    task_uid: Optional[str]
+    status: str
+    type: str
+    details: Optional[Dict[str, Any]]
+    duration: Optional[str]
+    enqueued_at: str
+    started_at: Optional[str]
+    finished_at: Optional[str]
```

Comment:
That's something you couldn't know, we decided it was best to have two different classes one for the task resource and another for all the asynchronous returns.
```
class Task(CamelBase):
    uid: str
    index_uid: str
    ...
    finished_at: Optional[str]

class TaskInfo(CamelBase):
    task_uid: str
    index_uid: str
    ...
    finished_at: Optional[str]
```